_data/security.yml

```diff
@@ -117,6 +117,15 @@ websites:
           text: "Only available for enterprise accounts."
       doc: https://engineering.opendns.com/2014/05/22/launching-two-step-verification/
 
+    - name: Pwnie Express Pwn Pulse
+      url: https://www.pwnieexpress.com/products/pulse-device-detection
+      img: pwnieexpress.png
+      tfa: Yes
+      software: Yes
+      exceptions:
+          text: "TFA Can be enabled on Pwn Pulse Accounts upon customer request."
+      doc: https://www.pwnieexpress.com/support
```

Comment:
There is information from that page under the Pulse -- User Guide link but it links to a CDN link which always feels a little unreliable to hardlink to : 

https://cdn2.hubspot.net/hubfs/421408/Pulse_User_Guide_Rev160630.pdf?t=1484262987343